### PR TITLE
[PULL REQUEST] Ch4 pseudo fluxes

### PR DESCRIPTION
**Pull request by rgryan92**
Activating pre-existing code in GEOS-Chem to read and write methane pseudo fluxes

### CODE REVIEW

**Comment by msulprizio** [comment]
This pull request impacts several blocks of code for `MODEL_GEOS`. I would defer to @lizziel on reviewing and merging these changes.

File: GeosCore/hcoi_gc_diagn_include.H [hunk 1]
```diff
@@ -27,6 +27,7 @@
   INTEGER, PARAMETER :: CATEGORY_NATURAL       = 3
   INTEGER, PARAMETER :: CATEGORY_BIOGENIC      = 4
   INTEGER, PARAMETER :: CATEGORY_BIOMASS       = 5
+  INTEGER, PARAMETER :: CATEGORY_ROCKET        = 6
```
Comment (lizziel) [inline GeosCore/hcoi_gc_diagn_include.H:30]: Is this change relevant to CH4 pseudo flux diagnostics?

Comment (lizziel) [inline GeosCore/hcoi_gc_diagn_include.H:30]: Is this change relevant to the CH4 pseudo flux update?

File: GeosUtil/gc_grid_mod.F90 [hunk 2]
```diff
@@ -134,7 +134,7 @@ SUBROUTINE Compute_Grid( Input_Opt, State_Grid, RC )
 
     ! Set maximum number of levels in the chemistry grid
     IF ( Input_Opt%LUCX ) THEN
-       State_Grid%MaxChemLev  = State_Grid%MaxStratLev
+       State_Grid%MaxChemLev  = State_Grid%NZ
```
Comment (lizziel) [inline GeosUtil/gc_grid_mod.F90:137]: Could explain why you made this change?

File: Headers/state_diag_mod.F90 [hunk 3]
```diff
@@ -174,7 +174,7 @@ MODULE State_Diag_Mod
      LOGICAL :: Archive_Loss
      LOGICAL :: Archive_Prod
 
-#ifdef MODEL_GEOS
+!#ifdef MODEL_GEOS
```
Comment (lizziel) [inline Headers/state_diag_mod.F90:177]: This change includes adding fields beyond what is needed for CH4 pseudo flux diagnostics. Is that intentional?

File: KPP/Standard/gckpp.map [hunk 4]
```diff
@@ -13,8 +13,8 @@
 !        R. Sander, Max-Planck Institute for Chemistry, Mainz, Germany
 ! 
 ! File                 : gckpp.map
-! Time                 : Thu Jun 11 18:03:10 2020
-! Working directory    : /n/home06/wdowns/GC/GCCcode/Code.12.9.0/KPP/Standard
+! Time                 : Tue Aug 25 12:43:27 2020
```
Comment (lizziel) [inline KPP/Standard/gckpp.map:16]: The KPP file changes indicate your base is out-of-date with current dev. Could you rebase?

File: log [hunk 5]
```diff
@@ -0,0 +1,215 @@
+make[1]: Entering directory `/lustre/home/ucfaea1/Models/GEOS-Chem/Codes/Code.12.9.3/GeosCore'
```
Comment (lizziel) [inline log:1]: I don't think this file was intended to be included in the PR.

File: run/GEOS/CodeDir [hunk 6]
```diff
@@ -0,0 +1 @@
+/home/ucfaea1/Models/GEOS-Chem/Codes/Code.12.9.3
```
Comment (lizziel) [inline run/GEOS/CodeDir:1]: Did you intend to change the run/GEOS files?

**Review by lizziel — CHANGES_REQUESTED** [review]

**Comment by lizziel** [comment]
Hi @rgryan92, I discussed this with Christoph Keller and he approved bringing it into the standard version. GMAO has been outputting it regularly in their real-time system and it seems to work fine. Rather than have you rebase I can go ahead and make a new PR with the modifications to take the diagnostic out of the MODEL_GEOS block.

**Pull request closed without merging**